recipe/laravel.php

```diff
@@ -35,6 +35,16 @@
     'storage/logs',
 ]);
 
+set('laravel_version', function () {
+    $result = run('{{bin/php}} {{release_path}}/artisan --version');
+
+    preg_match_all('/([0-9\.])$/', $result, $matches);
```

Comment:
Wrong, should be `'/([0-9.]+)/'`. You want to match ALL numbers/dots, not just 1. And it doesn't have to match the end of string.